schemas/zowe-yaml-schema.json

```diff
@@ -423,6 +423,35 @@
           "additionalProperties": false,
           "description": "Optional, advanced network configuration parameters",
           "properties": {
+            "bind": {
+              "type": "array",
+              "description": "The IP addresses which all of the Zowe servers will be binding to.",
+              "items": {
+                "type": "string"
+              }
+            },
+            "ciphers": {
+              "type": "array",
+              "description": "Acceptable TLS cipher suites for network connections.",
+              "items": {
+                "type": "string"
+              }
+            },
+            "curves": {
+              "type": "array",
+              "description": "Acceptable key exchange elliptic curves for network connections.",
+              "items": {
+                "type": "string"
+              }
+            },
+            "maxTls": {
+              "type": "string",
+              "description": "Maximum TLS version allowed for network connections. Must be: 1.0, 1.1, 1.2, or 1.3."
```

Comment:
if it 'must be' anything, make it an enum.
"enum": [ "1.0", "1.1" ]...
I agree with enum because "string" is unhelpful, it doesnt tell you what's possible.